docs/DEPLOY.md

```diff
@@ -0,0 +1,159 @@
+# Deployments with Stim
+
+Stim can be used to deploy to Kubernetes using Vault to configure a deployment environment.
+
```

Comment:
Every deployment needs many things, like passwords, to get started.  Stim deployment sets up a standard environment that will call the users' deployment script.